registry/registry.go

```diff
@@ -99,7 +101,11 @@ func doRequest(req *http.Request, jar http.CookieJar, timeout TimeoutType, secur
 			return false
 		}
 
-		hostDir := path.Join("/etc/docker/certs.d", req.URL.Host)
+		if runtime.GOOS == "windows" {
+			hostDir = filepath.Join(os.TempDir(), "/docker/certs.d", req.URL.Host)
```

Comment:
I think that's a good idea longer term - however, at the moment, the decision about where we put things is still a very open question which I am actively tracking (things will likely change in this regard when/if we docker daemon on Windows becomes a service and runs as localsystem). This change was the minimum bar to at least get the daemon running on Windows.